packages/kogito-db-migrator-tool/pom.xml

```diff
@@ -66,24 +66,32 @@
   <build>
     <testSourceDirectory>src/test/java</testSourceDirectory>
     <plugins>
-      <!-- Use mvn dependency:unpack command to unzip ddl files and mvn dependency:copy to copy into destination -->
       <plugin>
         <artifactId>maven-dependency-plugin</artifactId>
-        <configuration>
-          <artifactItems>
-            <artifactItem>
-              <groupId>org.kie.kogito</groupId>
-              <artifactId>kogito-ddl</artifactId>
-              <version>${version.org.kie.kogito}</version>
-              <classifier>db-scripts</classifier>
-              <type>zip</type>
-              <overWrite>true</overWrite>
-              <outputDirectory>src/main/resources</outputDirectory>
-              <destFileName>kogito-ddl-db-scripts.zip</destFileName>
-              <excludes>ansi/*</excludes>
-            </artifactItem>
-          </artifactItems>
-        </configuration>
+        <executions>
```

Comment:
This change is to allow the maven-dependency-plugin to run by default in the generate-sources phase. We are not using pnpm for this build so we will not invoke `mvn dependency:unpack` before the classical `maven deploy`